stm/asyncTaskQueue.ml

```diff
@@ -123,7 +123,7 @@ module Make(T : Task) = struct
                  "-worker-id"; name;
                  "-async-proofs-worker-priority";
                    Flags.string_of_priority !Flags.async_proofs_worker_priority]
-        | ("-ideslave"|"-emacs"|"-emacs-U"|"-batch")::tl -> set_slave_opt tl
```

Comment:
Ok but this it is the second time that I find a non-noop option removed by this PR, despite what the title claims. So I think it should be reviewed (I had only a very brief look).